src/spetlr/delta/delta_handle.py

```diff
@@ -112,10 +117,20 @@ def _validate(self):
             raise DeltaHandleInvalidFormat("Only format delta is supported.")
 
     def read(self) -> DataFrame:
-        """Read table by path if location is given, otherwise from name."""
+        reader = (
```

Comment:
from our communcation earlier - this should be removed. Since delta cannot encforce schema reading.